configurations/pipeline_configuration.py

```diff
@@ -269,10 +273,100 @@
                 ]
             ),
         ],
-        ws_correct_dataset_code_scheme=load_code_scheme("ws_correct_dataset")
-    ),
-    archive_configuration=ArchiveConfiguration(
-        archive_upload_bucket="gs://pipeline-execution-backup-archive",
-        bucket_dir_path="2022/RVI-ELECTIONS"
+        ws_correct_dataset_code_scheme=load_code_scheme("ws_correct_dataset"),
+        traffic_labels=[
+            # S01E01
+            # --------------------------------------- 12 - 13 March -----------------------------------------------                                       
+            TrafficLabel(isoparse("2022-03-12T00:00+03:00"), isoparse("2022-03-12T23:59+03:00"), "E01 Sat, Promo"),
+            TrafficLabel(isoparse("2022-03-13T00:00+03:00"), isoparse("2022-03-13T23:59+03:00"), "E01 Sun, Promo"),
+            # --------------------------------------- 14 March ---------------------------------------------------
+            TrafficLabel(isoparse("2022-03-14T00:00+03:00"), isoparse("2022-03-14T23:59+03:00"), "E01 Mon, None"),
+            # --------------------------------------- 15 March ---------------------------------------------------
+            TrafficLabel(isoparse("2022-03-15T00:00+03:00"), isoparse("2022-03-15T09:19+03:00"), "E01 Tue, None"),
+            TrafficLabel(isoparse("2022-03-15T09:20+03:00"), isoparse("2022-03-15T10:59+03:00"), "E01 Tue, Ad, Group A"),
```

Comment:
Can we add the total of the people we triggered the ad to here? You can pick this from textit flow runs after splitting consent withdrawn. 

```suggestion
            TrafficLabel(isoparse("2022-03-15T09:20+03:00"), isoparse("2022-03-15T10:59+03:00"), "E01 Tue, Ad, Group A- (104xxx)"),
```

Reply:
I have updated the labels.